pages/api/members/[id]/tasks.js

```diff
@@ -0,0 +1,30 @@
+const data = require('../../../../backend/data/index');
+
+module.exports = async (req, res) => {
+    await data.initIfNotStarted();
+    
+    if (req.method === 'POST') {
+        const authStatus = await data.auth.checkAnyUser(req.headers['authorization'], res);
```

Comment:
might be worth refactoring this out as middleware and wrapping as an higher order function (or I think Next.js has some support for middleware as well)

Reply:
(the authentication part)